src/bit-systems/object-menu.ts

```diff
@@ -38,6 +45,39 @@ function moveToTarget(world: HubsWorld, menu: EntityID) {
   setMatrixWorld(menuObj, targetObj.matrixWorld);
 }
 
+function openLink(world: HubsWorld, eid: EntityID) {
+  const { initialData } = createMessageDatas.get(eid)!;
+  const src = initialData.src;
+  // TODO: Currently only accounts for the simple case of an external url
+  //       but should support other type actions(eg: avatar update for avatar
+  //       url, room switch for Hubs room url).
+  //       See src/components/open-media-button.js
```

Comment:
Now that we aren't duplicated the object menus for every object, we could have each of these actions called from their own buttons if we wanted to, and then show/hide the buttons as needed. Otherwise, we'll need to set the text of the button for "use avatar" / "use scene" / "visit room" / "open link". It doesn't seem that important which way we choose, but since the only thing the buttons will have in common will be their position (not their text or functionality when clicked), I thought it might make sense to make them different buttons.